.ci/jobs/e2e-testing-mbp.yml

```diff
@@ -9,6 +9,7 @@
     scm:
       - github:
           branch-discovery: no-pr
+          head-filter-regex: '(master|7\.x|7\.11\.x|7\.10\.x)'
```

Comment:
@elastic/observablt-robots could it be possible that we skipped monitoring PRs with this change?

Reply:
yep, you'd need to add `PR-.*` see https://github.com/elastic/apm-integration-testing/blob/master/.ci/jobs/apm-integration-tests.yml#L14

Reply:
And tags?

Reply:
Sent a fix here: https://github.com/elastic/e2e-testing/pull/571. 🙏  👀